core-bundle/contao/library/Contao/Image.php

```diff
@@ -204,24 +240,9 @@ public static function getHtml($src, $alt='', $attributes='')
 				$darkSrc = substr($darkSrc, \strlen($webDir) + 1);
 			}
 
-			$darkAttributes = new HtmlAttributes($attributes);
-			$darkAttributes->mergeWith(array('class' => 'color-scheme--dark', 'loading' => 'lazy'));
-
-			foreach (array('data-icon', 'data-icon-disabled') as $icon)
```

Comment:
@Toflar you reintroduced #6869 in this one:
https://github.com/contao/contao/commit/3145d8b798c5191709cec250015b2ef9b1aa23e6